libnd4j/include/system/common.h

```diff
@@ -62,12 +62,12 @@
 #endif
 
 
-#ifdef __NEC__
+
 #include <unordered_map>
 #define SD_MAP_IMPL std::unordered_map
 #define SD_LOOPS_INLINED
 #define SD_INLINE inline

```

Comment:
Is this meant to become a default or something? Otherwise the contents of this ifdef should also be removed.